### PR TITLE
Add Join Table

### PR DESCRIPTION
**Pull request by danielufeli**
- Add New Column Vet, Visit
- Add New Data
- Add New Queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -42,3 +42,32 @@ ADD COLUMN owner_id INT,
...
     REFERENCES owners(id);
+
+/* Create a table named vets */
+
+CREATE TABLE vets (
+  id serial PRIMARY KEY,
+  name TEXT,
+  age INT,
+  date_of_graduation DATE
+);
```
Comment (CollinsTatang) [inline schema.sql]: - [x] Great job so far, please try to correct your datatype and add the `PRIMARY KEY` as seen below:
`CREATE TABLE vets (`
  ` id int NOT NULL IDENTITY(1,1),`
   `name varchar(255),`
   `age int,`
  ` date_of_graduation date,`
   `PRIMARY KEY (id)
   );
`

File: schema.sql [hunk 2]
```diff
@@ -42,3 +42,32 @@ ADD COLUMN owner_id INT,
...
+
+/* Many-to-many relationship between the tables species and vets */
+
+CREATE TABLE specializations (
+  vet_id INT,
+  species_id INT,
+  FOREIGN KEY (vet_id) REFERENCES vets(id),
+  FOREIGN KEY (species_id) REFERENCES species (id),
+  PRIMARY KEY (vet_id, species_id)
+);
```
Comment (CollinsTatang) [inline schema.sql]: - [x] Please this table should contain just the following columns:
`CREATE TABLE specializations (`
   `species_id int,`
  ` vet_id int
   );
`

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
Hi @danielufeli 👋, 

## STATUS: APPROVED ✅ 

You have done well implementing the required changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**